chart/values.yaml

```diff
@@ -427,6 +453,20 @@ scheduler:
   affinity: {}
   tolerations: []
 
+# Airflow migration and create user job settings
+jobs:
```

Comment:
We need to explicitly call out which job it is, so probably `createUserJob` would be more apt.

Reply:
Hi @kaxil, since the service account is used by both the create user job and the database migration job at this time, would you prefer to have that split into two separate accounts? I went with the generic `jobs` because of it mapping to both jobs that run during upgrade/install, but I'm good with splitting it out if desired.

Reply:
hmm.. 🤔 I am trying to think if in future we have more jobs that if name itself is self-descriptive it would be better

Reply:
Yea let's separate them, we also have a `cleanupJob`  -- while we just use `cleanup` name for it --- it is a bit confusing if we just have `jobs`